.changeset/spicy-plums-switch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-react": patch
+---
+
+Changed our internal dependency on `@floating-ui/react` from `^0.26.3` to `0.26.3` to prevent unexpected results when users use later patch versions of `@floating-ui/react` that may contain breaking behavioral changes.
```

Comment:
```suggestion
The dependency on `@floating-ui/react` has been temporarily pinned to `0.26.3` to avoid potential breaking behavioral changes from future versions.
```